src/components/navbar/index.cjsx

```diff
@@ -93,6 +94,7 @@ module.exports = React.createClass
             title={<UserName/>}
             ref='navDropDown'>
             {menuItems}
+            <AdminConsole/>
```

Comment:
oh I see. Could we just put the additional link in `src/components/navbar/username.cjsx` since it already listens to `CurrentUserStore`?